subt/artf_node.py

```diff
@@ -54,7 +54,10 @@ def check_results(result_mdnet, result_cv):
 
 
 def result2report(result, depth, fx):
-    # return relative XYZ distances to camera
+    """return relative XYZ distances to camera"""
+    if depth is None:
+        return None  # ignore detected artifacts for missing depth data
+                     # typically some glitch on start
```

Comment:
Should we maybe wait for the depth to be available like we do in main? Waiting for some necessary stuff.

Reply:
> Should we maybe wait for the depth to be available like we do in main? Waiting for some necessary stuff.

maybe the whole application should wait for initialization?

Reply:
Yeah, that is what I've arrived at as well.